app/scripts/views/mixins/account-locked-mixin.js

```diff
@@ -11,14 +11,14 @@
 define(function (require, exports, module) {
   'use strict';
 
-  var _ = require('underscore');
   var AuthErrors = require('lib/auth-errors');
   var BaseView = require('views/base');
+  var Cocktail = require('cocktail');
   var ResumeTokenMixin = require('views/mixins/resume-token-mixin');
 
   var t = BaseView.t;
 
-  var AccountLockedMixin = _.extend({
+  var AccountLockedMixin = Cocktail.mixin({
```

Comment:
This may cause side effects we aren't aware of, Cocktail is meant to be used with Views.